IdentityServer/v7/docs/content/reference/services/token_creation_service.md

```diff
@@ -28,8 +28,16 @@ for implementing the *ITokenCreationService* include modifying claims, audiences
 from a secondary data source, such as a profile service, database, or third-party service.
```

Comment:
Maybe just should cut this sentence on "Everyday use cases" - we shouldn't encourage this :)